net/wireshark3/Portfile

```diff
@@ -167,22 +168,37 @@ variant chmodbpf description {Enable Wireshark to access macOS capture devices}
     depends_run             port:wireshark-chmodbpf
 }
 
-variant python36 conflicts python37 python38 python39 description {Use python36 during build} {
-    depends_build-append    port:python36
-}
+set pythons_suffixes {36 37 38 39}
 
-variant python37 conflicts python36 python38 python39 description {Use python37 during build} {
-    depends_build-append    port:python37
+set pythons_ports {}
+foreach s ${pythons_suffixes} {
```

Comment:
Please use more descriptive variable names, like "pySuffix" (here), "pyVersion" below, that sort of thing.

Reply:
Done. I did my best to use the same variable names as in the rest of macports (`py_ver`, `py_ver_nodot`) and build from there

Reply:
Cool, thanks, and great idea on using the same var names used elsewhere.